### ISSUE TITLE
Create a setStatus function on the response object

### ISSUE THREAD

**Issue by designfrontier** [ansble/monument#271]
Create a setStatus method on the response object (connection.res within a route handler).
Use looks like any of the following:
```
connection.res.setStatus(404);
-or-
connection.res.setStatus('Not Found');
```

It should set the correct status message and status code using the https://nodejs.org/api/http.html#http_http_status_codes API

**Comment by xRahul** [comment]
Hey @designfrontier 

I'm gonna try this one

**Comment by designfrontier** [comment]
Awesome! Happy to help out with questions and support :-)

**Comment by xRahul** [comment]
@designfrontier Let me know if this is the correct way to implement it- https://github.com/ansble/monument/pull/272

**Comment by designfrontier** [comment]
Left you some notes... looking really good so far though

**Comment by xRahul** [comment]
@designfrontier made the changes as per coding style and will add tests later today!

**Comment by designfrontier** [comment]
@xRahul sorry didn'y have time to take a look last night on more then a surface level. Will have more time this evening to dig into it. Saw the undefined setStatus errors on the test you mentioned.

**Comment by designfrontier** [comment]
#283 merged this into the 2.5.x branch for release
